crates/forge/tests/integration/dispatchers.rs

```diff
@@ -180,6 +181,21 @@ fn handling_errors() {
                 }
             };
         }
+
+        #[test]
+        #[feature("safe_dispatcher")]
+        fn handling_missing_contract_error() {
```

Comment:
Cannot this message be asserted with `should panic` somehow? 🤔

Reply:
Should be possible, `should_panic` can take a String

Reply:
https://github.com/foundry-rs/starknet-foundry/issues/2114